.gitmodules

```diff
@@ -2,3 +2,7 @@
 	path = go-secp256k1-zkp
 	url = https://github.com/vulpemventures/go-secp256k1-zkp
 	branch = v1.1.6
+[submodule "regtest"]
+	path = regtest
+	url = https://github.com/BoltzExchange/regtest
+	branch = main
```

Comment:
Why a submodule and not `git clone` or `git pull` when needed? Feels like this is a risk of it becoming outdated easily